### PR TITLE
Use namespace var, instead of hardcoded value

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

Fixes #

## Proposed Changes

- use namespace vars for secret script

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: test/e2e-tests.sh [hunk 1]
```diff
@@ -351,9 +351,9 @@ function kafka_teardown() {
 
 function create_tls_secrets() {
   echo "Creating TLS Kafka secret"
-  STRIMZI_CRT=$(kubectl -n kafka get secret my-cluster-cluster-ca-cert --template='{{index .data "ca.crt"}}' | base64 --decode )
-  TLSUSER_CRT=$(kubectl -n kafka get secret my-tls-user --template='{{index .data "user.crt"}}' | base64 --decode )
-  TLSUSER_KEY=$(kubectl -n kafka get secret my-tls-user --template='{{index .data "user.key"}}' | base64 --decode )
+  STRIMZI_CRT=$(kubectl -n ${STRIMZI_KAFKA_NAMESPACE} get secret my-cluster-cluster-ca-cert --template='{{index .data "ca.crt"}}' | base64 --decode )
```
Comment (lionelvillard) [inline test/e2e-tests.sh:354]: where is it defined?

Comment (matzew) [reply test/e2e-tests.sh:354]: In the same script:
https://github.com/knative-sandbox/eventing-kafka/blob/744bdc1392718b060711c35b3aafe58da2c32eb9/test/e2e-tests.sh#L70

and used in other areas already.... e.g.:
https://github.com/knative-sandbox/eventing-kafka/blob/744bdc1392718b060711c35b3aafe58da2c32eb9/test/e2e-tests.sh#L314-L343

Comment (lionelvillard) [reply test/e2e-tests.sh:354]: thanks!

**Pull request merged**